packages/shared/src/components/modals/FeedFiltersModal.tsx

```diff
@@ -8,12 +8,20 @@ import useFeedSettings from '../../hooks/useFeedSettings';
 import { CustomSwitch } from '../fields/CustomSwitch';
 import { getFilterCardPreviews } from '../filters/FilterCardPreview';
 import { cloudinary } from '../../lib/image';
+import ThemeWidget from '../widgets/ThemeWidget';
 
 interface FeedFitlersModalProps extends ModalProps {
   trigger?: string;
 }
 
+const themes = [
```

Comment:
Perhaps we can TypeCast this, pretty sure we have a type for valid themes somewhere.
It's just to be more safe down the line.